lib/core/src/lib/layout/components/layout-container/layout-container.component.scss

```diff
@@ -37,6 +37,7 @@ adf-layout-container {
 #{$mat-sidenav-content},
 #{$mat-drawer-transition} #{$mat-drawer-content} {
     margin-left: 0 !important;
+    margin-right: 0 !important;
```

Comment:
Just to confirm - there is no other way to remove that space and adding !important in styles is required? (asking because we should to avoid !important)

Reply:
Unfortunately yes. The extra spacing being added was due to margin-right getting added on to the mat-sidenav-content element, via inline CSS. The only way I can think of to have a greater specificity than inline CSS is via the use of !important flag.